site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js

```diff
@@ -0,0 +1,14 @@
+const config = require('../config');
+
+/**
+ *
+ * @param {{context: {client: import('mongodb').MongoClient}}} context
+ */
+
+exports.up = async ({ context: { client } }) => {
+  await client.connect();
+
+  const reportsCollection = client.db(config.realm.production_db.db_name).collection('reports');
+
+  await reportsCollection.updateMany({}, { $set: { is_incident_report: true } });
```

Comment:
We assume all existing reports are linked to an incident. Is that correct?

Reply:
yes